tck/src/main/java/com/hedera/hashgraph/tck/methods/sdk/KeyService.java

```diff
@@ -100,14 +100,14 @@ private String processKeyRecursively(
                             .getPublicKey()
                             .toStringDER();
                 }
-                var publicKey = params.getType().equals(ED25519_PUBLIC_KEY)
-                        ? PrivateKey.generateED25519().getPublicKey().toStringDER()
-                        : PrivateKey.generateECDSA().getPublicKey().toStringDER();
+                var pKey = params.getType().equals(ED25519_PUBLIC_KEY)
```

Comment:
I suggest to rename this var to `privateKey`

Reply:
done